### DISCUSSION TITLE
fugitive.vim in LazyVim

### DISCUSSION THREAD

**Discussion by moonstruckmoth** [LazyVim/LazyVim#866] [Q&A] [answered]
I'm trying to use Tim Pope's fugitive.vim plugin. 
Some commands like Git status should open a new buffer which is interactive but in LazyVim, it is displaying a noice message which disappears after a few seconds.

Is there any way to have the default behavior for fugitive.vim?

TIA,
Raj

**Comment by dpetka2001** [comment, marked as answer]
It says in the docs that `Git` should be used in favor of `Gstatus` which was deprecated. If you use `Git` it opens in a split buffer in Lazyvim showing you the files that need to be staged or not just like `git status` normally shows.

**Reply by moonstruckmoth** [reply]
Thanks for clarifying this!
